packages/contracts/test/SignupAndPost.test.ts

```diff
@@ -111,13 +124,13 @@ describe('Unirep App', function () {
             // generate a fake proof
             const concoctProof = [...proof]
             const len = concoctProof[0].toString().length
-            concoctProof[0] = BigInt(
-                proof[0].toString().slice(0, len - 1) + BigInt(2)
-            )
+            concoctProof[0] = BigInt(2)
```

Comment:
What is the change here? Besides this, everything looks good

Reply:
The original concocted proof might create the same proof as the real one (but not 100% sure, the error was: `This Test should revert yet didn't`). I deem this change might lower the opportunity of collision.

Reply:
Make sense. It should lower the opportunity. But it's a bit weird, the original impl should be fine.